root/static/scripts/edit/URLCleanup.js

```diff
@@ -2540,6 +2539,19 @@ const CLEANUPS: CleanupEntries = {
       };
     },
   },
+  'eonkyo': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?e-onkyo\\.com', 'i')],
+    restrict: [LINK_TYPES.downloadpurchase],
```

Comment:
Actually, you want to restrict linking `/album/` to releases only:

```suggestion
    restrict: [
      {release: LINK_TYPES.downloadpurchase.release},
    ],
```

Reply:
Thanks, resolved in [291c650](https://github.com/metabrainz/musicbrainz-server/pull/3328/commits/291c650fd49f969f1eaa4a58020244a027474488) — I'd assumed that restricting to `LINK_TYPES.downloadpurchase` implied it could be used for releases only based on the edits I've put in myself, but now that I look closer, I see that the relationship also exists for other entity types 🤦